ReSharper.FSharp/src/FSharp.Psi.Intentions/src/QuickFixes/UpdateLiteralConstantFix.fs

```diff
@@ -0,0 +1,84 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open FSharp.Compiler.Symbols
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Psi.Tree
+open JetBrains.ReSharper.Resources.Shell
+
+type UpdateLiteralConstantFix(error: LiteralConstantValuesDifferError) =
+    inherit FSharpQuickFixBase()
+    let errorRefPat = error.Pat.As<IReferencePat>()
+    
+    let tryFindDeclarationFromSignature () =
+        let containingTypeDecl = errorRefPat.GetContainingTypeDeclaration()
+        let decls = containingTypeDecl.DeclaredElement.GetDeclarations()
+        decls |> Seq.tryFind (fun d -> d.GetSourceFile().IsFSharpSignatureFile)
+        
+    let tryFindSigBindingSignature sigMembers =
+        let p = errorRefPat.Binding.HeadPattern.As<IFSharpPattern>()
+        if Seq.length p.Declarations = 1 then
+            let implDec = Seq.head p.Declarations
+            let declName = implDec.DeclaredName
+            sigMembers
+            |>  Seq.tryPick(fun m ->
+                let bindingSignature = m.As<IBindingSignature>()
+                match bindingSignature with
+                | null -> None
+                | _ ->
+                    match bindingSignature.HeadPattern with
+                    | :? IReferencePat as sigRefPat when
+                        declName = sigRefPat.DeclaredName -> Some bindingSignature
+                    | _ -> None
+                )
+        else
+            None
+
+    let mutable sigRefPat = null
+
+    let rec isImplExprValidInSig (implExpression: IFSharpExpression) =
+        match implExpression with
+        | :? IReferenceExpr as refExpr ->
+            refExpr.Reference.ResolveWithFcs(sigRefPat, System.String.Empty, false, refExpr.IsQualified)
```

Comment:
Please pass something like `UpdateLiteralConstantFix.IsAvailable` as the `opName`.